Classes/common/CDTIncrementalStore.m

```diff
@@ -27,7 +27,7 @@
 #import "CDTISGraphviz.h"
 
 #ifdef HAS_NSBatchUpdateRequest
-/**
```

Comment:
Documentation looks great, but I'm wondering why the asterisks, like this one, were taken out, the smart documentation popup doesn't show anymore when I Option+click a method or parameter.